azuredeploy.json

```diff
@@ -240,11 +240,11 @@
             ],
             "defaultValue": "S1",
             "metadata": {
-                "description": "MS SQL  database trasaction units"
+                "description": "MS SQL database service object names. There are a lot more than S* (to be added later). See https://docs.microsoft.com/en-us/rest/api/sql/databases/createorupdate#ServiceObjectiveName"
```

Comment:
This description doesn't make sense here. Having more to come is irrelevant to someone using this template as they cannot add these others without forking and editing. Should this be an issue?

Reply:
Yeah, I knew this didn't look good, and yet, I felt bad that not all options were specified. Not sure if it'll make sense to enumerate all of them anyway. Removed.